go-runtime/ftl/call_overrider.go

```diff
@@ -0,0 +1,21 @@
+package ftl
+
+import (
+	"context"
+)
+
+type CallOverrider interface {
```

Comment:
I didn't want this to be public to module code, but wasn't sure of a better place to put it...